### DISCUSSION TITLE
Unable to verify the domain for my github pages website

### DISCUSSION THREAD

**Discussion by ludiocel** [community/community#68291] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I own the domain "ludiocel.xyz" and have a website with Github Pages, to verify my domain it asks me to make a TXT record with a certain name and value, which i've done **several** times correctly, when I press verify however it says that it couldn't be verified and to wait up to 24 hours (which i've waited a lot more, i've been trying since the 2nd of September).

**Comment by milindaShehan** [comment]
**Double-check DNS records:**
Go to your domain registrar's website (the service where you bought your domain, like GoDaddy, Namecheap, etc.).
Log in to your account and find the DNS management or DNS settings section.
Make sure you have created a TXT record with the correct name (this will usually be provided by GitHub) and value (a long string of characters provided by GitHub for verification).

**Check propagation:**
Sometimes, DNS changes can take time to spread across the internet. It's usually fast, but in rare cases, it might take longer than 24 hours. Be patient and try again later.
Clear DNS cache:

This is about clearing your computer's memory of DNS information it might have stored.

> **For Windows:**

Open the Command Prompt with administrative privileges (right-click and choose "Run as administrator").
Type `ipconfig /flushdns` and press Enter.

> **For macOS and Linux:**

Open a terminal window.
Type ` sudo dscacheutil -flushcache` and press Enter. You might be asked for your administrator password.

**Check for any conflicting DNS settings:**

Make sure that you haven't set up conflicting DNS settings with your domain registrar or any other hosting service you might be using. For example, if you're using a different service for email hosting, make sure it's not interfering.

**Contact GitHub Support:**

If none of the above steps work, you might need to reach out to GitHub Support. They have a dedicated support team that can provide specific assistance for issues like this.

**Consider a different verification method:**

Some services might offer alternative methods for domain verification, such as adding a specific DNS record type (like CNAME or A record). Check if GitHub Pages provides alternative verification methods.
